config/contents/style/word_array.md

```diff
@@ -2,4 +2,27 @@ This cop can check for array literals made up of word-like
 strings, that are not using the %w() syntax.
 
 Alternatively, it can check for uses of the %w() syntax, in projects
-which do not want to include that syntax.
\ No newline at end of file
+which do not want to include that syntax.
+
+Configuration option: MinSize
+If set, arrays with fewer elements than this value will not trigger the
+cop. For example, a `MinSize of `3` will not enforce a style on an array
```

Comment:
Looks like there's a character missing here, a \` right after `MinSize`

Reply:
Thanks Marc.

This content is generated from Rubocop docs. Please submit an issue/PR to Rubocop repo as well.

Reply:
Done in https://github.com/bbatsov/rubocop/pull/4834